### PR TITLE
Documentation for the ppx.

### PR DESCRIPTION
**Pull request by Drup**
I decided not to change the tutorial much because I think there are enough to understand about the type system (which is the main subject of the tutorial) and it would not be beneficial to present the syntax too.

### CODE REVIEW

File: doc/manual-wiki/intro.wiki [hunk 1]
```diff
@@ -30,6 +30,7 @@ The documentation for TyXML combinators is provided in <<a_api|module Html5_sigs
 The first thing to understand about TyXML is that for most intents and purposes, it is exactly like Html. As such, the [[https://developer.mozilla.org/en-US/docs/Web/HTML/Element|Html reference]] is still very useful. For each Html elements or attributes, there is a combinator implementing it. The main differences are that you can use OCaml to manipulate the elements and that invalid Html produces a type error.
 
 In this tutorial, we will build the [[https://github.com/ocsigen/tyxml/tree/master/examples/mini_website|Mini website]].
+If you prefer the native Html syntax, you can also use the <<a_manual chapter="ppx"|ppx syntax extension>> and consult the [[https://github.com/ocsigen/tyxml/tree/master/examples/mini_website_ppx|ppx mini website]].
```
Comment (vasilisp) [inline doc/manual-wiki/intro.wiki]: (No important comments, just nitpicking.)

Capitalize HTML and SVG everywhere?

File: doc/manual-wiki/ppx.wiki [hunk 2]
```diff
@@ -0,0 +1,83 @@
+= TyXML Ppx syntax extension =
+
+TyXML provides a ppx to allow writing of Html5 and Svg documents
+in a native syntax while still enjoying the benefits and type-safety of TyXML.
```
Comment (vasilisp) [inline doc/manual-wiki/ppx.wiki]: to allow <del>of</del> writing <del>of</del> HTML and SVG documents _in the standard syntax_

File: doc/manual-wiki/ppx.wiki [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+For the ppx to be available, you need to install ##markup## and ##ppx_tools##.
+<<code|
+opam install markup ppx_tools tyxml
+>>
+
+==@@id="syntax"@@ Syntax ==
+
+The ppx provides two quotations, ##html5## and ##svg## (along with disambiguated versions, ##tyxml.html5## and ##tyxml.svg##.
```
Comment (vasilisp) [inline doc/manual-wiki/ppx.wiki]: What the disambiguated versions do is a bit ambiguous :).

**Comment by Drup** [comment]
@vasilisp Thanks! Should be fixed.

File: doc/manual-wiki/intro.wiki (hunk 1 above)
Comment (Drup) [inline doc/manual-wiki/intro.wiki]: I need to do that at some point, yes ...

**Pull request merged**